nucliadb_protos/writer.proto

```diff
@@ -71,18 +71,9 @@ message BrokerMessage {
     // Field Conversations
     map<string, resources.Conversation> conversations = 11;
 
-    // Field Layout
-    map<string, resources.FieldLayout> layouts = 12;
-
     // Field Text
     map<string, resources.FieldText> texts = 13;
 
-    // Field keyword
-    map<string, resources.FieldKeywordset> keywordsets = 14;
-
-    // Field Datetime
-    map<string, resources.FieldDatetime> datetimes = 15;
-
```

Comment:
Deprecate this fields instead of removing them. Although we are doing a new nucliadb version, we shouldn't be doing a breaking change, we should be Bw/c

Reply:
Sorry, is not a breaking change. But could you reserve the numbers so we don't accidentally reuse them?